libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -1,39 +1,70 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import * as path from 'path';
+import { JsonFile, Import, IPackageJson } from '@rushstack/node-core-library';
+import { BaseFlag } from './base/BaseFlag';
 
-import { FileSystem, JsonFile, JsonObject, Import } from '@rushstack/node-core-library';
-
-import { PackageManagerName } from './packageManager/PackageManager';
-import { RushConfiguration } from './RushConfiguration';
+import type { PackageManagerName } from './packageManager/PackageManager';
+import type { RushConfiguration } from './RushConfiguration';
 
 const lodash: typeof import('lodash') = Import.lazy('lodash', require);
 
 export const LAST_INSTALL_FLAG_FILE_NAME: string = 'last-install.flag';
 
+/**
+ * This represents the JSON data structure for the "last-install.flag" file.
+ * @internal
+ */
+export interface ILastInstallFlagJson {
+  /**
+   * Current node version
+   */
+  node: string;
```

Comment:
```suggestion
  nodeVersion: string;
```

Reply:
Hi, @iclanton . Do you need to rename these fields in this PR? It seems that most fields like "node", "packageJson", "storePath", "workspaces" are existing fields in current last install flag.